plugin/vcs/vcs.go

```diff
@@ -24,6 +24,17 @@ func (e Type) String() string {
 	return "UNKNOWN"
 }
 
+// OAuthToken is the API message for OAuthToken.
+type OAuthToken struct {
+	AccessToken  string `json:"access_token" `
+	RefreshToken string `json:"refresh_token"`
+	ExpiresIn    int64  `json:"expires_in"`
+	CreatedAt    int64  `json:"created_at"`
+	// ExpiresTs is a derivative from ExpresIn and CreatedAt.
+	// ExpiresTs = ExpiresIn == 0 ? 0 : CreatedAt + ExpiresIn
+	ExpiresTs int64 `json:"expires_ts"`
```

Comment:
For ExpiresTs, maybe we don't need json annotation since it's NOT unmarshaled from the JSON response.

Reply:
OAuthToken is a standard, so we can move it to common/oauth.go

Reply:
the json annoation added here is for naming consistent. The frontend still need this field, and if we leave the annotation blank, the data received by the frontend would have multiple naming style.